src/com/shop/Shop.java

```diff
@@ -261,33 +261,34 @@ public static void main(String[] args) {
 				}
 			} while (!shop.isCustomerVerified());
 
-			while (!menuOption.equalsIgnoreCase("x")) {
+			while (menuOption != 'x') {
 				shop.displayMenu();
-				menuOption = Character.toString(shop.input.next().charAt(0));
+				// make users input lower case after entry to then have to only check for lowercase input
+				menuOption = Character.toLowerCase(shop.input.next().charAt(0));
```

Comment:
Makes users input lowercase after entry, makes us only have to analyze lowercase options